nemo-python/tests/test.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+import csv
+import os
+import math
+import sys
+
+from nmo_python import load_string, load_file, NemoEngine, NemoOutputManager
+
+class TestStringMethods(unittest.TestCase):
+    def test_reason(self):
+        rules = """
+        data(1,2) .
+        data(hi,42) .
+        data(hello,world) .
+
+        calculated(?x, !v) :- data(?y, ?x) .
+        """
+
+        engine = NemoEngine(load_string(rules))
+        engine.reason()
+
+        result = list(engine.result("calculated"))
+
+        expected_result = [
+            ['2', '<__Null#9223372036854775809>'],
+            ['42', '<__Null#9223372036854775810>'],
+            ['world', '<__Null#9223372036854775811>']]
+
+        self.assertEqual(result, expected_result)
+
+def stringify(value):
```

Comment:
Please make this file conform to PEP-8 (which, among other things, requires two blank lines between top-level definitions), ideally by just using https://github.com/psf/black.